### DISCUSSION TITLE
Directly open Enterprise login page, and populate fields

### DISCUSSION THREAD

**Discussion by bveldkamp** [EsriPS/arcgis-assistant-feedback#51] [Ideas]
For convenience, I'd like to add a hyperlink on our Enterprise's home page that opens the Enterprise login page for Assistant, and populates the URL and App ID fields. 

Would this be possible by adding appropriate URL parameters?

**Comment by jpeterson** [comment]
Hi @bveldkamp! This is not currently possible, but it's a great idea! Thanks for sharing.
